client/lib/mixins/i18n/index.js

```diff
@@ -14,7 +14,7 @@ var debug = require( 'debug' )( 'calypso:i18n' ),
 var config = require( 'config' ),
 	numberFormatPHPJS = require( './number-format' ),
 	emitter = require( 'lib/mixins/emitter' ),
-	interpolateComponents = require( 'lib/interpolate-components' );
+	interpolateComponents = require( 'lib/interpolate-components/src' );
```

Comment:
Weird I just tested and it didn't work. Would be really nice to require the lib like any other external project would do!